### ISSUE TITLE
Create Hyperlinks Automatically (Thunderbird)

### ISSUE THREAD

**Issue by Yaripped69** [fnxweb/urllink#16]
Would it be possible to turn text links into clickable links in Thunderbird?

Thank you

**Comment by fnxweb** [comment]
It's not something I'm _currently_ planning, but I'll leave this open in case I ever have time.
Are you talking about having it happen automatically?

**Comment by nischwartz** [comment]
This is a function that I've been looking for - I get emails from systems and due to the encoding, the link shows as plain text, not a hyperlink.  Is this posible for the extension to automatically convert those links?

**Comment by fnxweb** [comment]
Yes, it's theoretically possible, but I currently have zero code in the extension for actual modifications to the email's DOM, the work is all done on the fly from the right-click and selection.  I'd have to hook into the window update to pre-parse the whole email (presuming I can) when one is selected and fiddle with the DOM to make the text a link.

Not a small task.

**Comment by nischwartz** [comment]
Figured it would be trickier than just adding an option!  I'm guessing it's not a highly requested addition to TB, so don't go out of your way just for me!  Thanks for the quick reply!

**Comment by fnxweb** [comment]
To be fair, it's probably one of the _highest_ requested features (given I get a req. once a year or so!).

But I have little enough time any more to maintain this, and quite a few bit and bobs put put back in that used to be there before the Firefox WebExtensions rewrite (keyboard shortcuts and the like, now they're back in FF/TB), plus a bug to do with TB settings and opening new windows I still need to debug.

Ho hum...
